### PR TITLE
Move from Lager to Logger

### PR DESCRIPTION
**Pull request by ichernetsky**
JIRA issue: [DCOS_OSS-5461](https://jira.mesosphere.com/browse/DCOS_OSS-5461)

### CODE REVIEW

**Comment by urbanserj** [comment]
@ichernetsky could you please remove lager configuration from `./erldns.config.example` and create another PR for upstream? I hope all our changes will be eventually merged to dnsimple/erldns.

**Review by urbanserj — APPROVED** [review]

File: src/erldns_app.erl [hunk 1]
```diff
@@ -19,37 +19,39 @@
...
 
-  lager:info("Notifying servers to start"),
+  ?LOG_INFO("Notifying servers to start"),
   erldns_events:notify(start_servers),
 
   ok.
 
 stop(_State) ->
-  lager:info("Stop erldns application"),
+  ?LOG_INFO("Stop erldns application"),
```
Comment (GoelDeepak) [inline src/erldns_app.erl:54]: Now that we are here, the start of the erldns is at DEBUG level and stop is at INFO level. Maybe we could move start also to INFO level.

Comment (urbanserj) [reply src/erldns_app.erl:54]: That will be challenging to convince upstream maintainers to merge such patch.

Comment (GoelDeepak) [reply src/erldns_app.erl:54]: don't we use our own fork?

Comment (urbanserj) [reply src/erldns_app.erl:54]: We do, but I almost always push patches to upstream. It take up to a year to get them merge.

**Comment by ichernetsky** [comment]
@urbanserj, I replaced the Lager config in it with a logger one. Please take a look.

**Pull request merged**